spec/controllers/imports_controller_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe ImportsController, type: :controller do
+  let(:user) { create(:user) }
+  let(:import_artist_attributes) { { name: 'Raul Seixas' } }
+
+  before do
+    sign_in(user)
+  end
+
+  describe "POST #artist" do
+    before do
+      VCR.use_cassette('controllers/import/import_raul_seixas') do
+        post :artist, params: { import_artist: import_artist_attributes }
+      end
+    end
+
+    it 'creates an Artist called Raul Seixas' do
+      expect(Artist.where(name: 'Raul Seixas').first.name).to eq('Raul Seixas')
+    end
+  end
+
+  describe "POST #artist_releases" do
+    let(:artist) { Artist.first }
+    let(:import_artist_releases_attributes) {
+      { artist_id: artist.id, import_artist_releases: { max_results: 15 } }
+    }
+
+    before do
+      Artist.new(name: 'Raul Seixas', user: user).save
+
+      VCR.use_cassette('controllers/import/import_raul_seixas_releases') do
+        post :artist_releases, params: import_artist_releases_attributes
+      end
+    end
+
+    it 'creates 15 releases for the artist called Raul Seixas' do
+      expect(Artist.where(name: 'Raul Seixas').first.releases.count).to eq(15)
+    end
+
+    it "creates a release called 'Eu Nasci Há 10 Mil Anos Atrás' for Raul Seixas user" do
```

Comment:
Single quotes :nail_care: 

Also:

```suggestion
    it "creates a release called 'Eu Nasci Há 10 Mil Anos Atrás' for Raul Seixas artist" do
```